### ISSUE TITLE
MIssing atoms in bondi_radii dict for VdwVolumeABC

### ISSUE THREAD

**Issue by martinwk** [mordred-descriptor/mordred#103]
description
--
I want to calculate the van der waals volume with the `VdwVolumeABC` descriptor. However, when I have Smiles with an element that is not in the `bondi_radii` dictionary in the file `mordred\VdwVolumeABC.py`, the descriptor yields an error. I wondered why not all elements are present in `bondi_radii` dict.

```
from rdkit import Chem
Chem.MolFromSmiles('CC1=CC(=O)N=[S](O1)([O-])=O.[K+]')
--

### OS/distribution

windows 10

### conda or pip

conda

### python version

3.10

```.bash
python --version
```

### library version

Please execute the command and paste result.
```
# Name                    Version                   Build  Channel
alabaster                 0.7.12                     py_0    conda-forge
appdirs                   1.4.4                    pypi_0    pypi
apptools                  5.1.0              pyh44b312d_0    conda-forge
apscheduler               3.9.1.post1      py39hcbf5309_0    conda-forge
astroid                   2.12.12          py39hcbf5309_1    conda-forge
attrs                     22.1.0             pyh71513ae_1    conda-forge
autopep8                  2.0.0              pyhd8ed1ab_0    conda-forge
babel                     2.11.0             pyhd8ed1ab_0    conda-forge
boost                     1.78.0           py39hbd792c9_4    conda-forge
boost-cpp                 1.78.0               h9f4b32c_1    conda-forge
brotli                    1.0.9                hcfcfb64_8    conda-forge
brotli-bin                1.0.9                hcfcfb64_8    conda-forge
brotlipy                  0.7.0           py39ha55989b_1005    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2022.9.24            h5b45459_0    conda-forge
cairo                     1.16.0            h63a05c6_1001    conda-forge
certifi                   2022.9.24          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1           py39h68f70e3_2    conda-forge
charset-normalizer        2.1.1              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
configobj                 5.0.6                      py_0    conda-forge
contourpy                 1.0.6            py39h1f6ef14_0    conda-forge
cryptography              38.0.3           py39hb6bd5e6_0    conda-forge
curl                      7.86.0               heaf79c2_1    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
dill                      0.3.6              pyhd8ed1ab_1    conda-forge
docutils                  0.17.1           py39hcbf5309_3    conda-forge
double-conversion         3.2.0                h63175ca_1    conda-forge
eigen                     3.4.0                h2d74725_0    conda-forge
envisage                  6.0.1              pyhd8ed1ab_0    conda-forge
esbonio                   0.14.3                   pypi_0    pypi
et_xmlfile                1.0.1                   py_1001    conda-forge
exceptiongroup            1.0.4              pyhd8ed1ab_0    conda-forge
expat                     2.5.0                h1537add_0    conda-forge
fipy                      3.4.3            py39hcbf5309_0    conda-forge
fonttools                 4.38.0           py39ha55989b_1    conda-forge
freetype                  2.12.1               h546665d_0    conda-forge
future                    0.18.2             pyhd8ed1ab_6    conda-forge
gettext                   0.21.1               h5728263_0    conda-forge
gl2ps                     1.4.2                h0597ee9_0    conda-forge
glew                      2.1.0                h39d44d4_2    conda-forge
glib                      2.74.1               h12be248_1    conda-forge
glib-tools                2.74.1               h12be248_1    conda-forge
greenlet                  2.0.1            py39h99910a6_0    conda-forge
gst-plugins-base          1.18.5               he07aa86_3    conda-forge
gstreamer                 1.18.5               hdff456e_3    conda-forge
hdf4                      4.2.15               h1b1b6ef_5    conda-forge
hdf5                      1.12.2          nompi_h57737ce_100    conda-forge
icu                       58.2                 ha925a31_3
idna                      3.4                pyhd8ed1ab_0    conda-forge
imagesize                 1.4.1              pyhd8ed1ab_0    conda-forge
importlib-metadata        5.0.0              pyha770c72_1    conda-forge
importlib_metadata        5.0.0                hd8ed1ab_1    conda-forge
importlib_resources       5.10.0             pyhd8ed1ab_0    conda-forge
iniconfig                 1.1.1              pyh9f0ad1d_0    conda-forge
intel-openmp              2022.1.0          h57928b3_3787    conda-forge
isort                     5.10.1             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
joblib                    1.2.0              pyhd8ed1ab_0    conda-forge
jpeg                      9e                   h8ffe710_2    conda-forge
jsoncpp                   1.9.4                h2d74725_3    conda-forge
jsonpickle                2.2.0                    pypi_0    pypi
kiwisolver                1.4.4            py39h1f6ef14_1    conda-forge
krb5                      1.19.3               hc8ab02b_0    conda-forge
lazy-object-proxy         1.8.0            py39ha55989b_0    conda-forge
lcms2                     2.14                 h90d422f_0    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libblas                   3.9.0              16_win64_mkl    conda-forge
libbrotlicommon           1.0.9                hcfcfb64_8    conda-forge
libbrotlidec              1.0.9                hcfcfb64_8    conda-forge
libbrotlienc              1.0.9                hcfcfb64_8    conda-forge
libcblas                  3.9.0              16_win64_mkl    conda-forge
libclang                  12.0.1          default_h81446c8_4    conda-forge
libcurl                   7.86.0               heaf79c2_1    conda-forge
libdeflate                1.14                 hcfcfb64_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libflang                  5.0.0           h6538335_20180525    conda-forge
libglib                   2.74.1               he8f3873_1    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
liblapack                 3.9.0              16_win64_mkl    conda-forge
libnetcdf                 4.9.0           nompi_h85765be_100    conda-forge
libogg                    1.3.4                h8ffe710_1    conda-forge
libpng                    1.6.38               h19919ed_0    conda-forge
libsqlite                 3.40.0               hcfcfb64_0    conda-forge
libssh2                   1.10.0               h9a1e1f7_3    conda-forge
libtheora                 1.1.1             h8d14728_1005    conda-forge
libtiff                   4.4.0                h8e97e67_4    conda-forge
libvorbis                 1.3.7                h0e60522_0    conda-forge
libwebp-base              1.2.4                h8ffe710_0    conda-forge
libxcb                    1.13              hcd874cb_1004    conda-forge
libxml2                   2.9.14               hf5bbc77_4    conda-forge
libzip                    1.9.2                h519de47_1    conda-forge
libzlib                   1.2.13               hcfcfb64_4    conda-forge
llvm-meta                 5.0.0                         0    conda-forge
loguru                    0.6.0            py39hcbf5309_2    conda-forge
lz4-c                     1.9.3                h8ffe710_1    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
markupsafe                2.1.1            py39ha55989b_2    conda-forge
matplotlib                3.6.2            py39hcbf5309_0    conda-forge
matplotlib-base           3.6.2            py39haf65ace_0    conda-forge
mayavi                    4.7.2            py39h342a29d_5    conda-forge
mccabe                    0.7.0              pyhd8ed1ab_0    conda-forge
mkl                       2022.1.0           h6a75c08_874    conda-forge
mordred                   1.2.0                      py_1    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
networkx                  2.8.8              pyhd8ed1ab_0    conda-forge
numpy                     1.23.4           py39hbccbffa_1    conda-forge
openjpeg                  2.5.0                hc9384bd_1    conda-forge
openmp                    5.0.0                    vc14_1    conda-forge
openpyxl                  3.0.10           py39ha55989b_2    conda-forge
openssl                   3.0.7                hcfcfb64_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pandas                    1.1.5            py39h2e25243_0    conda-forge
patsy                     0.5.3              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                h17e33f8_0    conda-forge
periodictable             1.5.2                      py_0    conda-forge
pillow                    9.2.0            py39h595c93f_3    conda-forge
pint                      0.20.1             pyhd8ed1ab_0    conda-forge
pip                       22.3.1             pyhd8ed1ab_0    conda-forge
pixman                    0.38.0            hfa6e2cd_1003    conda-forge
platformdirs              2.5.2              pyhd8ed1ab_1    conda-forge
plotly                    4.14.3                     py_0    plotly
plotly-orca               1.3.1                         1    plotly
pluggy                    1.0.0              pyhd8ed1ab_5    conda-forge
proj                      6.2.1                ha7a8c7b_0    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
pubchempy                 1.0.4                    pypi_0    pypi
pycairo                   1.21.0           py39h82915f6_2    conda-forge
pycodestyle               2.9.1              pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pydantic                  1.10.2                   pypi_0    pypi
pyface                    7.4.3              pyhd8ed1ab_0    conda-forge
pygls                     0.13.0                   pypi_0    pypi
pygments                  2.13.0             pyhd8ed1ab_0    conda-forge
pyiast                    1.5.1                    pypi_0    pypi
pylint                    2.15.5             pyhd8ed1ab_0    conda-forge
pyopenssl                 22.1.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pyqt                      5.15.4           py39h415ef7b_0    conda-forge
pyqt5-sip                 12.9.0           py39h415ef7b_0    conda-forge
pysocks                   1.7.1              pyh0701188_6    conda-forge
pyspellchecker            0.7.0                    pypi_0    pypi
pytest                    7.2.0              pyhd8ed1ab_2    conda-forge
python                    3.9.13          hcf16a7b_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.9                      2_cp39    conda-forge
pytz                      2022.6             pyhd8ed1ab_0    conda-forge
qspr                      0.1                      pypi_0    pypi
qt-main                   5.15.2               he8e5bd7_7
rdkit                     2022.09.1        py39h6c30a6c_1    conda-forge
reportlab                 3.5.68           py39h12d40c5_1    conda-forge
requests                  2.28.1             pyhd8ed1ab_1    conda-forge
retrying                  1.3.3                      py_2    conda-forge
scikit-learn              1.1.3            py39h6fe01c0_1    conda-forge
scipy                     1.9.3            py39hfbf2dce_2    conda-forge
seaborn                   0.12.1               hd8ed1ab_0    conda-forge
seaborn-base              0.12.1             pyhd8ed1ab_0    conda-forge
setuptools                59.8.0           py39hcbf5309_1    conda-forge
sip                       6.5.1            py39h415ef7b_2    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
sklearn-json              0.1.0                    pypi_0    pypi
snowballstemmer           2.2.0              pyhd8ed1ab_0    conda-forge
sphinx                    5.3.0              pyhd8ed1ab_0    conda-forge
sphinx_rtd_theme          1.1.1              pyhd8ed1ab_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    2.0.0              pyhd8ed1ab_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.5              pyhd8ed1ab_2    conda-forge
sqlalchemy                1.4.44           py39ha55989b_0    conda-forge
sqlite                    3.40.0               hcfcfb64_0    conda-forge
statsmodels               0.13.5           py39hc266a54_2    conda-forge
tbb                       2021.7.0             h91493d7_0    conda-forge
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tk                        8.6.12               h8ffe710_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
tomlkit                   0.11.6             pyha770c72_0    conda-forge
tornado                   6.2              py39ha55989b_1    conda-forge
tqdm                      4.64.1             pyhd8ed1ab_0    conda-forge
traits                    6.4.1            py39ha55989b_1    conda-forge
traitsui                  7.4.2              pyhd8ed1ab_0    conda-forge
typeguard                 2.13.3                   pypi_0    pypi
typing                    3.10.0.0           pyhd8ed1ab_0    conda-forge
typing-extensions         4.4.0                hd8ed1ab_0    conda-forge
typing_extensions         4.4.0              pyha770c72_0    conda-forge
tzdata                    2022f                h191b570_0    conda-forge
tzlocal                   2.1                pyh9f0ad1d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
unicodedata2              15.0.0           py39ha55989b_0    conda-forge
urllib3                   1.26.11            pyhd8ed1ab_0    conda-forge
utfcpp                    3.2.2                h57928b3_0    conda-forge
vc                        14.3                 h3d8a991_9    conda-forge
vs2015_runtime            14.32.31332          h1d6e394_9    conda-forge
vtk                       9.0.3           py39h84a6d72_201
wheel                     0.38.4             pyhd8ed1ab_0    conda-forge
win32_setctime            1.1.0              pyhd8ed1ab_0    conda-forge
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
wrapt                     1.14.1           py39ha55989b_1    conda-forge
wt-process-models         0.3                       dev_0    <develop>
xorg-libxau               1.0.9                hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
zipp                      3.10.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               hcfcfb64_4    conda-forge
zstd                      1.5.2                h7755175_4    conda-forge
```
